src/platform/converters.ts

```diff
@@ -1,13 +1,39 @@
 import byFamily from "../generated/platformAdapter";
 
-import type { Account, CryptoCurrency, Transaction } from "../types";
+import type {
+  Account,
+  CryptoCurrency,
+  TokenAccount,
+  TokenCurrency,
+  Transaction,
+} from "../types";
 import type {
   PlatformAccount,
   PlatformCurrency,
   PlatformTransaction,
 } from "./types";
 
-export function accountToPlatformAccount(account: Account): PlatformAccount {
+export function accountToPlatformAccount(
+  account: Account | TokenAccount,
```

Comment:
What about the case with `ChildAccount`? I see `AccountLike[]` is passed to this function on desktop side which includes child account type. I believe `ChildAccount` doesn't have `freshAddress` field so I guess `parentAccount.freshAddress` should be passed instead? 🤔